src/Processors/QueryPlan/ReadFromMergeTree.h

```diff
@@ -19,6 +19,9 @@ class Pipe;
 
 using MergeTreeReadTaskCallback = std::function<std::optional<ParallelReadResponse>(ParallelReadRequest)>;
 
+using PartitionIdToMaxBlock = std::unordered_map<String, Int64>;
```

Comment:
This is also defined in MergeTreeData.h. Not sure what's the original one between that one and Context.h. Ok either way